tools/generators/service/typed-json-config/tsconfig.json.ts

```diff
@@ -0,0 +1,20 @@
+import { TsConfig } from '../../types';
+
+export const packageTsConfig: TsConfig = {
+  extends: '../../tsconfig.json',
+  compilerOptions: {
+    preserveSymlinks: true,
+    baseUrl: '.',
+    esModuleInterop: true,
+  },
+  references: [
+    { path: '../../packages/configuration' },
+    { path: '../../packages/serverless-configuration' },
+    { path: '../../packages/serverless-helpers' },
```

Comment:
As we have seen, we need to point to a different config.
```suggestion
    { path: '../../packages/serverless-helpers/tsconfig.build.json' },
```
Could also make the change in `backend/core/tsconfig.json`, in a different commit ?